test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/DataAnnotationsModelValidatorTest.cs

```diff
@@ -220,55 +220,6 @@ public void ValidateReturnsMemberNameIfItIsDifferentFromDisplayName()
 #endif
 
         [Fact]
-        public void GetClientValidationRules_ReturnsEmptyRuleSet()
```

Comment:
where are the replacements for these tests?  don't see a `DataAnnotationsClientModelValidatorTest` class.